protractor-docker/docker-selenium-grid-conf.js

```diff
@@ -0,0 +1,19 @@
+exports.config = {
+	framework: 'jasmine2'
+	, specs: ['specs/*.js']
+ 	, seleniumAddress: 'http://localhost:4444/wd/hub'
+	,  capabilities: {
+		browserName: 'chrome'
+		, shardTestFiles: true
+		, maxInstances: 5
+	}
+	, getPageTimeout: 180000
+	, allScriptsTimeout: 180000
+	, onPrepare: function () {
+		browser.manage().window().maximize();
```

Comment:
Add comment about why `onPrepare` we need to maximize the window. This could be necessary if a user is running debug and wants to view the website... also add a comment about `implicitylyWait`.